test/python/transpiler/test_1q.py

```diff
@@ -74,7 +74,7 @@ def test_device(self, circuit, level):
         dsc="Transpiling {circuit.__name__} at level {level} should work for simulator",
         name="{circuit.__name__}_level{level}_valid",
     )
-    def test_simulator(self, circuit, level):
+    def basic_simulator(self, circuit, level):
```

Comment:
Removing the `test_` prefix will cause this test to be skipped, right?

Reply:
oversight from a search and replace